pallets/utxo/src/weights.rs

```diff
@@ -40,4 +40,9 @@ impl<T: frame_system::Config> crate::WeightInfo for WeightInfo<T> {
             .saturating_add(T::DbWeight::get().reads(3 as Weight))
             .saturating_add(T::DbWeight::get().writes(3 as Weight))
     }
+
+    fn token_create(u: u32) -> Weight {
+        // Under construction
+        (u as Weight).saturating_add((100 * 1_000 * 100_000_000 as Weight))
```

Comment:
Ah this is where the confusion has come from with the 100 vs 100 * 1_000 * 100_000_000 the fee should be the big one. I think we can leave the weight as it was before

Reply:
Would it be correct? 

```rust
(u as Weight).saturating_add((100 as Weight))
```

Reply:
Yep I think so, the weight is the tx weight rather than the fee so there's no need for it to be massive

Reply:
Done